safety_plan.Rmd

```diff
@@ -8,8 +8,8 @@ output:
 links-to-footnotes: false
 paged-footnotes: false
 params:
-  repo_url: 'https://github.com/NewGraphEnvironment/mybookdown-template/'
-  report_url: 'https://newgraphenvironment.github.io/mybookdown-template'
+  repo_url: 'https://github.com/NewGraphEnvironment/safety_plan_peace_2024'
```

Comment:
idea behind updateing these was to use them in place of objects defined in the setup chunk (ex. `params$repo_url` swapped into code in place of `link_repo`).  Its basically just another way of doing the same thing.  Can try on future projects